modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/DisasterRecoveryManager.java

```diff
@@ -575,6 +576,15 @@ private static Map<TablePartitionId, GlobalPartitionState> assembleGlobal(
                     return assembleGlobalStateFromLocal(catalog, tablePartitionId, map);
                 }));
 
+        if (partitionIds.isEmpty()) {
```

Comment:
Why don't we do this 'make missing partitions unavailable' trick if we are given explicit partition IDs?